checkov/circleci_pipelines/image_referencer/provider.py

```diff
@@ -13,26 +15,34 @@ def __init__(self, workflow_config: dict[str, Any], file_path: str) -> None:
         self.file_path = file_path
         self.workflow_config = workflow_config
 
+    def generate_resource_key(self, start_line: int, end_line: int, tag: str) -> str:
+        sub_name = resolve_sub_name(self.workflow_config, start_line, end_line, tag)
+        image_name = resolve_image_name(self.workflow_config[tag][sub_name], start_line, end_line)
+        new_key = f'{tag}.{sub_name}.docker.image#{image_name}' if sub_name else tag
```

Comment:
```suggestion
        new_key = f'{tag}.{sub_name}.docker.image:{image_name}' if sub_name else tag
```
Why not a colon? That's the convention Im familiar with and we use in other places

Reply:
changed :)